go.mod

```
@@ -1,13 +1,12 @@
 module github.com/np-guard/vpc-network-config-synthesis
 
-go 1.22.4
+go 1.23
```

```suggestion
go 1.23.0
```
I'm getting errors without the bugfix version